src/widget/wlibrarytableview.cpp

```diff
@@ -260,3 +260,35 @@ void WLibraryTableView::focusInEvent(QFocusEvent* event) {
         }
     }
 }
+
+QModelIndex WLibraryTableView::moveCursor(CursorAction cursorAction,
+        Qt::KeyboardModifiers modifiers) {
+    // By default the home and end keys move to the first and last column rather
+    // than the first and last row
```

Comment:
I had to read that a few times to figure you describe what is _overridden_, not the new implementation. Let's rephrase it:
```suggestion
    // Make the home and end keys move to the first and last row rather than
    // the first and last column (QAbstractItemView default)
```

Reply:
Sure, I'll change that. Btw, I'm not the only one who does this ha. if you grep for `By default` you'll get 46 other matches.